pkgs/tools/misc/q/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenvNoCC, fetchFromGitHub, python2 }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "q";
```

Comment:
No single letter names. `q-text-as-data` instead.

Reply:
Where is this policy stated? There seemt o be other packages (eg. `j`) with single letter names.

Reply:
ping @FRidh

Reply:
`j` was added very long ago, when Nixpkgs was also still tiny by comparison.
This policy has not been documented and falls in my opinion under common sense.

Reply:
Huh, okay.

Reply:
There is also `t`, added in more recently in 2015.

 I can change the name but I do feel like it would make it harder to find the package: when I first checked if q had already been package I did not check `q-text-as-data` or similar.

Reply:
Considering it is a single letter it is in general hard to find. At least with `q-text-as-data` it's [in line with other distro's](https://repology.org/project/q-text-as-data/versions).